stripe/src/main/java/com/stripe/android/model/PaymentIntentParams.java

```diff
@@ -139,7 +223,7 @@ public PaymentIntentParams setReturnUrl(@NonNull String returnUrl) {
      * @return {@code this}, for chaining purposes
      */
     @NonNull
-    public PaymentIntentParams setExtraParams(@NonNull Map<String, Object> extraParams) {
+    public PaymentIntentParams setExtraParams(@Nullable Map<String, Object> extraParams) {
```

Comment:
I don't think this should be null either

Reply:
How are extra params used? Are there certain required params?

Reply:
oof - thanks for questioning me on this, I think I misread.